lib/engine.js

```diff
@@ -87,13 +83,7 @@ const getTitles = (packages) => {
   return titles;
 };
 
-const getFilter = (argv) => {
-  if (argv.f !== undefined) {
-    return argv.f;
-  }
-
-  return '*';
-};
+const getFilter = argv => ((argv.f !== undefined) ? argv.f : '*');
```

Comment:
I think that this can be further simplified to:

```javascript
const getFilter = argv => argv.f || '*';
```

This assumes that an empty string should become `'*'`.